clang/unittests/Format/TokenAnnotatorTest.cpp

```diff
@@ -1489,11 +1489,25 @@ TEST_F(TokenAnnotatorTest, RequiresDoesNotChangeParsingOfTheRest) {
 TEST_F(TokenAnnotatorTest, UnderstandsAsm) {
   auto Tokens = annotate("__asm{\n"
                          "a:\n"
-                         "};");
-  ASSERT_EQ(Tokens.size(), 7u) << Tokens;
+                         ": x\n"
+                         ":};");
```

Comment:
Is this real/valid assembly code?

Reply:
It's valid. The input, output, and clobber operands are a comma-separated list of C variables (or equivalently produced CPP macros), which may be empty.